sktime/transformations/series/dropna.py

```diff
@@ -0,0 +1,239 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Transformer to drop rows or columns containing missing values."""
+
+__author__ = ["hliebert"]
+
+from sktime.transformations.base import BaseTransformer
+
+
+class DropNA(BaseTransformer):
+    """Drop missing values transformation.
+
+    Drops rows or columns with missing values from X. Mostly wraps
+    pandas.DataFrame.dropna, but allows specifying thresh as a fraction of
+    non-missing observations.
+
+    Parameters
+    ----------
+    axis : {0 or 'index', 1 or 'columns'}, default 0
+        Determine if rows or columns which contain missing values are removed.
+        Must be 0 or 'index' for univariate input.
+
+        * 0, or 'index' : Drop rows which contain missing values.
+        * 1, or 'columns' : Drop columns which contain missing value.
+
+    how : {'any', 'all'}, default 'any'
+        Determine if row or column is removed from DataFrame, when we have
+        at least one NA or all NA.
+
+        * 'any' : If any NA values are present, drop that row or column.
+        * 'all' : If all values are NA, drop that row or column.
+
+    thresh : int or float, optional
+         If int, require that many non-NA values (as in pandas.dropna). If
+         float, share of non-NA values for rows or columns to be retained.
```

Comment:
Will it make sense to add "at least" for `int` case? Just in case someone misinterpretes it to be exact value.

For `float` case, I found this line a bit difficult to follow. Is `retained` the intended word, because it sounds like `thresh=0.3` mean 30% non-null values will be kept and rest will be removed?

Reply:
Yes, that seems much better to me.

Can you please also mention about being strict proper fractions, 0/1 exclusive part?

Reply:
I've adjusted it as discussed.